Unit testing on github is failing with `'ietf/name/models.py:104: error: Need type annotation for "next_states"  [var-annotated]'`. Did something change in the settings?

--- a/ietf/name/fixtures/names.json
+++ b/ietf/name/fixtures/names.json
@@ -3034,7 +3034,9 @@
       "material_types": "[\n    \"slides\"\n]",
       "matman_roles": "[\n    \"chair\"\n]",
       "need_parent": false,
-      "parent_types": [],
+      "parent_types": [
+        "rfcedtyp"
+      ],
       "req_subm_approval": true,
       "role_order": "[\n    \"chair\"\n]",
       "session_purposes": "[\n    \"regular\"\n]",

